execution/utility.sh

```diff
@@ -1819,13 +1819,11 @@ function cleanup_elbv2_rules() {
 # -- S3 --
 
 function isBucketAccessible() {
-  local region="$1"; shift
   local bucket="$1"; shift
   local prefix="$1"; shift
 
-  local result_file="$(getTopTempDir)/is_bucket_accessible_XXXXXX.txt"
-
-  aws --region ${region} s3 ls "s3://${bucket}/${prefix}${prefix:+/}" > "${result_file}" 2>&1
+  result_file="$(getTopTempDir)/is_bucket_accessible_XXXXXX.txt"
+  aws s3 ls "s3://${bucket}/${prefix}${prefix:+/}" > "${result_file}" 2>&1
```

Comment:
Presume S3 CLI doesn't need the region?

Reply:
Yeah since S3 is a global service it seems to ignore the region for the s3 services. It accepts fine, but just less places we need to pass the region.

Reply:
thought that might be the case